debian_radius/bionic/privacyidea-radius.postinst

```diff
@@ -12,10 +12,7 @@ if [ -f /usr/share/dbconfig-common/dpkg/postinst.pgsql  ]; then
 fi
 
 activate_perl() {
-  if [ -L /etc/freeradius/3.0/mods-enabled/perl ]; then
-    rm /etc/freeradius/3.0/mods-enabled/perl
-  fi
-  ln -s /etc/freeradius/3.0/mods-available/mods-perl-privacyidea /etc/freeradius/3.0/mods-enabled/perl
+  ln -s /etc/freeradius/3.0/mods-available/privacyidea-perl /etc/freeradius/3.0/mods-enabled/
```

Comment:
if we would call 

    ln -sf /etc/freeradius/3.0/mods-available/privacyidea-perl /etc/freeradius/3.0/mods-enabled/

two times it would fail. But it looks like we did it this way before.
Obviously it was only run once, since this is only called during the ``configure)`` step. So it did not break.

If we ever move the ``activate_perl`` and other ``activate_`` out of the ``configure)`` step we have to watch out, that it does not break.
I think there is no change needed now. But if you want to, you could change it to:

   ln -sf ...